### PR TITLE
Add Walnut and goat-cheese recipe #17

### PR DESCRIPTION
**Pull request by wikmag**
Pasta recipe with leeks, walnut and goat cheese.

### CODE REVIEW

File: pasta/Walnut-goat-cheese.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Ingredientss
```
Comment (JosseBergUiB) [inline pasta/Walnut-goat-cheese.md:1]:
```suggestion
## Ingredients
```

**Comment by samumantha** [comment]
Sounds delicious! Do you still want to add more instructions to this recipe?

**Comment by wikmag** [comment]
I think the recipe is complete.

**Comment by rkdarst** [comment]
thank you!

**Pull request merged**